docs/blog-articles.md

```diff
@@ -1,4 +1,5 @@
+* [Memory Leaks in Android](https://www.raywenderlich.com/4690472-memory-leaks-in-android)
 * [Detect memory leaks in your instrumentation tests using LeakCanary](https://proandroiddev.com/detecting-memory-leaks-in-your-instrumentation-tests-using-leakcanary-1268e911d5ce)
 * [9 ways to avoid memory leaks in Android](https://android.jlelse.eu/9-ways-to-avoid-memory-leaks-in-android-b6d81648e35e)
 
-Your article should be here, please update this list! Any technical level welcome.
\ No newline at end of file
+Your article should be here, please update this list! Any technical level welcome.
```

Comment:
Do you know why it's highlighting this line as changed? Line end symbol changed?